### PR TITLE
Create Freegridco.in.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
create

### CODE REVIEW

File: Freegridco.in/Freegridco.in.fc [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+
+        if (IsVisible(ElementByXPath ("//input[@name='login']") )) 
+	    {
+         var box_user = ElementByXPath("//input[@name='login']");
+         SetText(box_user, GetSetting("Email"));
+        }
+		
+		Wait(2);
+		
+        if (IsVisible(ElementByXPath ("//input[@name='password']") ))
```
Comment (faucetcollector) [inline Freegridco.in/Freegridco.in.fc]: Best to do the elementbyxpath only once. Also if you dont log or exit then you dont need the if statement at all, as the settext will not do anything if the element is not found

File: Freegridco.in/Freegridco.in.fc [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+	{
+        ad=false; //solve the problem in start - after he return to script again
+        //return GetWaitSetting();  
+
+        var WaitTimerText = ElementByXPath("//p[@id='roll_wait_text']");	
+        if (IsVisible(WaitTimerText))
+        {
+             Log("FaucetClaim: WaitTimerText apper"); 
+             int min,sec;
+             min = Convert.ToInt32(WaitTimerText.Text.Trim().Replace("Wait for", "").Replace("before next roll", "").Split(':')[0])*60;
```
Comment (faucetcollector) [inline Freegridco.in/Freegridco.in.fc]: Please use int.TryParse

File: Freegridco.in/Freegridco.in.fc [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+             sec = Convert.ToInt32(WaitTimerText.Text.Trim().Replace("Wait for", "").Replace("before next roll", "").Split(':')[1]);
```
Comment (faucetcollector) [inline Freegridco.in/Freegridco.in.fc]: Best to put the result of the .Split into a variable and use that to get the sec and min calculated

Comment (aviramdayan) [reply Freegridco.in/Freegridco.in.fc]: yeah also way, but it work, in the next time, tnx for the tip

**Pull request merged**